docs/source/reference/northstar/callbacks.rst

```diff
@@ -87,22 +87,25 @@ lobby\_private_lobby.gnut:
 .. cpp:function:: bool ClientCommandCallback_PrivateMatchSetPlaylistVarOverride( entity player, array<string> args )
 .. cpp:function:: bool ClientCommandCallback_ResetMatchSettingsToDefault( entity player, array<string> args )
 
-mp\_ai_mp.gnut:
+mp\\_ai_mp.gnut:
 ^^^^^^^^^^^^^^^
 
 .. cpp:function:: bool SPMP_Callback_ForceAIMissPlayer( entity npc, entity player )
 
-mp\_base_gametype_mp.gnut:
+mp\\_base_gametype_mp.gnut:
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. cpp:function:: bool ClientCommandCallback_spec_next( entity player, array<string> args )
 .. cpp:function:: bool ClientCommandCallback_spec_prev( entity player, array<string> args )
 .. cpp:function:: bool ClientCommandCallback_spec_mode( entity player, array<string> args )
 
-mp\_gamestate_mp.nut:
+mp\\_gamestate_mp.nut:
 ^^^^^^^^^^^^^^^^^^^^^
 
 .. cpp:function:: void AddCallback_OnRoundEndCleanup( void functionref() callback )
 .. cpp:function:: void SetTimeoutWinnerDecisionFunc( int functionref() callback )
 
-    
\ No newline at end of file
+ui\\ui_mouse_capture.nut:
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. cpp:function:: void AddMouseMovementCaptureHandler( var capturePanelOrMenu, void functionref( int deltaX, int deltaY ) func )
```

Comment:
```suggestion
.. cpp:function:: void AddMouseMovementCaptureHandler( var capturePanelOrMenu, void functionref( int deltaX, int deltaY ) func )

    Fires every time the mouse is moved while the MouseMovementCapture is being clicked. `deltaX` and `deltaY` are the difference in pixels the mouse moved since the last callback fired.
```

Reply:
Tbh I'm not sure where this comment on the functionality should be? Other functions dont have any comments explaining them in this fashion?